client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,144 @@
+import React, { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './FeedbackForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  subject: Yup.string().required('Required'),
+  name: Yup.string().required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  problem_description: Yup.string().required('Required')
+});
+
+const FeedbackForm = ({ authenticatedUser }) => {
+  const location = useLocation();
+  const dispatch = useDispatch();
+  const isAuthenticated = authenticatedUser != null;
+  const creating = useSelector(state => state.ticketCreate.creating);
+  const creatingError = useSelector(state => state.ticketCreate.creatingError);
+  const creatingErrorMessage = useSelector(
+    state => state.ticketCreate.creatingErrorMessage
+  );
+  const creatingSuccess = useSelector(
+    state => state.ticketCreate.creatingSuccess
+  );
+  const createdTicketId = useSelector(
+    state => state.ticketCreate.createdTicketId
+  );
+
+  const url = location.pathname;
+  const defaultValues = useMemo(
+    () => ({
+      subject: 'Feedback',
```

Comment:
That seems useful, and matches requirement no. 4

> The form should submit: (page/s) user visited before the form

Please, interpret pages as "route/s" (because this is a web app).

@jchuahtacc has provided great insight into what helps the support team, so if one needs an approved definition of how to send this data, one could ask him.

Reply:
> Should we note in the subject in what area they are looking at by adding the route? Like "Feedback (history/jobs)" or "Feedback portal (history/jobs)". Would that be useful? Or we could prepend or append the route information to whatever they have entered (in `problem_description")?
> 
> Like for example:
> `subject`: "Feedback (history/jobs)"
> and then also we prepend something to the problem description during the dispatch
> `problem_description`: + "Feedback for portal (history/jobs)"

On creating Feedback/Ticket the route is saved as HTTP_REFERER, in this case I think we don't need any additional metadata/field.

Reply:
And is the route/`HTTP_REFERER` available to the ticket recipient (i.e. support staff)?

Reply:
It's part of ticket information itself and it's shown in the email sent and on consult.tacc.utexas.edu.

Reply:
For Subject, I asked @thbrowntacc, subject for now will be "Feedback for the Frontera Portal"